src/main/java/org/fundacionjala/coding/melvi/movierefactor/Customer.java

```diff
@@ -0,0 +1,50 @@
+package org.fundacionjala.coding.melvi.movierefactor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ */
+public class Customer implements ICustomer {
+    private String name;
+    private List<Rental> rentals = new ArrayList<Rental>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)